tests/integration/test_repair_tools.py

```diff
@@ -0,0 +1,227 @@
+""""Testing of repair tools."""
+
+from ansys.geometry.core.modeler import Modeler
+from ansys.geometry.core.tools.repair_tool_message import RepairToolMessage
+
+
+def test_find_split_edges(modeler: Modeler):
+    """Test if problem areas are detectable."""
+    modeler.open_file("./tests/integration/files/SplitEdgeDesignTest.scdocx")
```

Comment:
Is it possible to create one test where we run a repair tool on geometry that was _only_ created in pygeom, instead of loading an scdocx? It would be nice to have one test like that

Reply:
@umutsoysalansys let me know if this is impossible